deadmatterconfig.json

```diff
@@ -0,0 +1,128 @@
+[
+    {
+        "DisplayName":"Steam Server Name",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets the server name",
+        "Keywords":"server,name,steamservername",
+        "FieldName":"SteamServerName",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"SteamServerName",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"AMP Powered Server",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Lobby Header",
```

Comment:
Why isn't this setting using the same FieldName. Isn't it just the server name? Usually SteamServerName is used to display the server name in the Steam browser, which sometimes the normal server name does not do

Reply:
The ServerName in the Game.ini displays in the lobby (but not the server list) when you join a server. SteamServerName in the previous parameter only sets the name in the server browser, and not the lobby. The two are not linked.

Reply:
I get that they display in different places, but wouldn't they normally be the same? As you have put in as default values? In which case have them both set by the same FieldName, and one of the settings is hidden

Reply:
Ah ok, that makes sense. Like this?
```json
{
        "DisplayName":"Lobby Header",
        "Category":"Dead Matter Server Settings",
        "Description":"Sets the server title displayed in the lobby",
        "Keywords":"server,name,lobby,servername",
        "FieldName":"SteamServerName",
        "Hidden":true,
        "InputType":"text",
        "IsFlagArgument":false,
        "ParamFieldName":"/Script/DeadMatter.DMGameSession.ServerName",
        "IncludeInCommandLine":false,
        "EnumValues":{}
    }
```

Reply:
Yup, but remove the keywords, otherwise the hidden setting will be displayed in search, which is nonsensical because the user can't see it

Reply:
Updated in latest commit